pages/tutorials/3.0/graphics/vertex-array.md

```diff
@@ -460,11 +460,15 @@ private:
 
     void resetParticle(std::size_t index)
     {
+        // create random number generator
+        static std::random_device rd;
+        static std::mt19937       rng(rd());
```

Comment:
Isn't it bad practice to recreate the device and RNG for every iteration/index?

Reply:
yes it is

Reply:
Note those variables are created once because `static`

Reply:
The random number generator is only constructed once then is reused on all subsequent calls.